### ISSUE TITLE
Assign is being referenced from @ember/polyfills. This prevents upgrade to Ember 4

### ISSUE THREAD

**Issue by cah-brian-gantzler** [san650/ember-cli-page-object#571]
Assign is being exported here https://github.com/san650/ember-cli-page-object/blob/master/addon-test-support/-private/helpers.js#L1. Why would we be exporting assign? This is being transpiled to below. This is preventing an upgrade to ember 4.
Could we remove this import please

![image](https://user-images.githubusercontent.com/1529286/162498613-23027aa9-9cd7-4faf-943b-f07b1ba47189.png)

(Updated. Thought it was an import, turns out to be an export, which is even weirder)

**Comment by cah-briangantzler** [comment]
Ok I see that this assign is imported in many other files. Could this line be replaced with?
```js
export { Object.assign as assign };
```

**Comment by cah-briangantzler** [comment]
Appears this is fixed in PR https://github.com/san650/ember-cli-page-object/pull/536/files but this was merged into the V2-beta branch

Since this branch hasnt been touched since Jan 30, any way we could maybe backport this fix so consumers can upgrade to Ember 4 without waiting on the V2 version of this addon?

**Comment by ro0gr** [comment]
Sorry for delay. 

Yes, I'd prefer to use the object spread approach as you referred above. Unfortunately, I doubt if it can be just cherry-picked, cause of many changes in beta happened and conflicts are possible. I think we still can do it manually though, w/o a big issue.

**Comment by ro0gr** [comment]
Just released the fix as a part of https://github.com/san650/ember-cli-page-object/releases/tag/v1.17.12
